src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -37,8 +37,8 @@
 from toil.job import Job
 
 from toil_scripts.adam_uberscript.automated_scaling import SparkMasterAddress
-from toil_scripts.batch_alignment.bwa_alignment import docker_call
-from toil_scripts.spark_utils.spawn_cluster import start_spark_hdfs_cluster
+from toil_scripts.batch_alignment.bwa_alignment import docker_call, mock_mode
+from toil_scripts.spark_utils.spawn_cluster import *
```

Comment:
Agreed, but that change was due to some problem with how imports work with Toil. @fnothaft said using ```*``` was necessary.